externs/shaka/player.js

```diff
@@ -896,6 +897,8 @@ shaka.extern.ManifestConfiguration;
  *   If true, if the protocol is HTTP change it to HTTPs.
  * @property {boolean} preferNativeHls
  *   If true, prefer native HLS playback when possible, regardless of platform.
+ * @property {number} updateIntervalSeconds
+ *   The minimum number of seconds to see if the manifest has changes.
```

Comment:
This is fine for now, but you're really making me wonder if we should have an event instead.  The manifest parser could send an event (through the player, to keep streaming decoupled) any time it updates.  It might be a little weird with the `updateEvery()` method we use for SegmentTemplate+duration, but once we have a mechanism for closing SegmentIndexes that we aren't using (thanks, @michellezhuogg!), that event would only fire once per media type (audio + video + maybe text).